### DISCUSSION TITLE
HELP!! Doesn't work!

### DISCUSSION THREAD

**Discussion by Sam-hacks1** [3kh0/ext-remover#1226] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
The ExtHang3r unblocker thing only works for like 2 and a half fucking seconds, then it turns off.

**Comment by ingoau** [comment]
Please make your title more descriptive...

**Comment by akabutnicer** [comment, marked as answer]
First of all, this only works if your Chromebook doesn't have an exceptional CPU. Second of all, why are you still using extension hangers?

**Reply by Sam-hacks1** [reply]
it works but only for a little bit and what else should I use other than extension hangers?
